templates/namerd-admin-service.yaml

```diff
@@ -2,22 +2,16 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: namerd
+  name: namerd-admin
   namespace: infra
-{{- if .Values.service.annotations }}
+{{- if .Values.namerdAdmin.service.annotations }}
   annotations:
-{{ toYaml .Values.service.annotations | indent 4 }}
+{{ toYaml .Values.namerdAdmin.service.annotations | indent 4 }}
 {{- end }}
 spec:
   selector:
     app: namerd
-  type: LoadBalancer
+  type: {{ .Values.namerdAdmin.service.type }}
   ports:
-    - name: thrift
-      port: 4100
-    - name: http
-      port: 4180
-    - name: mesh
-      port: 4321
-    - name: admin
-      port: 9991
+  - name: admin
+    port: 9991
```

Comment:
Completely drop this service. I created it, initially, to give a different type to just the admin interface. I'm not sure we need this in a public chart, at all.